src/OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -21,9 +23,21 @@ namespace orbit_base {
 
 #ifdef _WIN32
 
+[[nodiscard]] inline uint64_t GetPerformanceCounterPeriodNs() {
+  LARGE_INTEGER frequency;
+  QueryPerformanceFrequency(&frequency);
+  return 1'000'000'000 / frequency.QuadPart;
+}
+
+[[nodiscard]] inline uint64_t RawTimestampToNs(uint64_t raw_timestamp) {
```

Comment:
This needs a bit of explanation, especially now that it's in `OrbitBase`: What is a raw timestamp? Where does it come from, i.e., when would you use such function?

Reply:
Added a comment.

Reply:
"Raw timestamps" is  how Microsoft refers to the value of the performance counter, in ETW anyways. For example, to setup ETW to produce events which timestamps are the result of QueryPerformanceCounter, we need to specify `PROCESS_TRACE_MODE_RAW_TIMESTAMP`.

From https://docs.microsoft.com/en-us/windows/win32/api/evntrace/ns-evntrace-event_trace_logfilea : 
"By default, ProcessTrace converts the event's TimeStamp from the original raw format (system time, QPC time, or CPU cycle counter) into system time (100-nanosecond intervals since midnight, January 1, 1601).

Specify the PROCESS_TRACE_MODE_RAW_TIMESTAMP flag if you do not want the time stamp value in the TimeStamp member of EVENT_HEADER and EVENT_TRACE_HEADER converted to system time. If this flag is specified, ProcessTrace will leave the time stamp value in the original format that the controller specified in the Wnode.ClientContext member of EVENT_TRACE_PROPERTIES."

I see your point, but I think it is better to keep as is to be coherent with Microsoft's nomenclature. This function is used to convert our ETW "raw" timestamps to nanoseconds, RawTimestampToNs() seems appropriate. Using that same function in "CaptureTimestampNs()" documents the link between QueryPerfomanceCounter() and Microsoft's concept of a "raw timestamp".

Reply:
From the documentation you cite there are multiple possible raw timestamps, and the one used by ETW is the one specified with `Wnode.ClientContext`. So the name is more generic than what the function is doing: it converts from one specific raw timestamp, the performance counter, but there are more.

Also the official Microsoft name is clearly "performance counter" if it's returned by a function called "query performance counter"...

Reply:
Ah right, I read the doc wrong. Will open a PR.